mysite/account/tests.py

```diff
@@ -107,21 +117,23 @@ def test_logout_no_open_redirect(self):
 
 
 class ProfileGetsCreatedWhenUserIsCreated(WebTest):
-
-    """django-authopenid only creates User objects, but we need Person objects
-    in all such cases. Test that creating a User will automatically create
-    a Person in our project."""
+    """
+    Test that a user profile is created when a user  is created.
```

Comment:
@ehashman Remind me if I ever write a book to have you edit it. ;)  Will fix this.